content/en/blog/_posts/2023-08-15-retroactive-default-storage-class-ga.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Retroactive Default StorageClass move to GA"
+date: 2023-08-15
```

Comment:
/hold

I want to get it confirmed that this is the data that release comms have assigned. OK to unhold if it is.

Reply:
Hi, release lead shadow for 1.28 here. 

According to the [tracking board](https://github.com/orgs/kubernetes/projects/140/views/6) the blog has been scheduled for August 18th 2023. Please update the date accordingly, thanks!